Makefile

```diff
@@ -46,14 +51,21 @@ $(BUILD_DIR)/linux-amd64/$(BINARY_NAME):
 $(BUILD_DIR)/windows-amd64/$(BINARY_NAME).exe:
 	CGO_ENABLED=0 GOARCH=amd64 GOOS=windows go build -ldflags="$(LDFLAGS)" -o $@ $(GO_BUILDFLAGS) ./cmd/admin-helper/
 
+$(BUILD_DIR)/macos-universal/$(BINARY_NAME): $(BUILD_DIR)/macos-amd64/$(BINARY_NAME) $(BUILD_DIR)/macos-arm64/$(BINARY_NAME) $(TOOLS_BINDIR)/makefat
+	mkdir -p $(BUILD_DIR)/macos-universal
+	cd $(BUILD_DIR) && $(TOOLS_BINDIR)/makefat macos-universal/$(BINARY_NAME) macos-amd64/$(BINARY_NAME) macos-arm64/$(BINARY_NAME)
+
 .PHONY: cross ## Cross compiles all binaries
-cross: $(BUILD_DIR)/macos-amd64/$(BINARY_NAME) $(BUILD_DIR)/macos-arm64/$(BINARY_NAME)-arm64 $(BUILD_DIR)/linux-amd64/$(BINARY_NAME) $(BUILD_DIR)/windows-amd64/$(BINARY_NAME).exe
+cross: $(BUILD_DIR)/macos-amd64/$(BINARY_NAME) $(BUILD_DIR)/macos-arm64/$(BINARY_NAME) $(BUILD_DIR)/linux-amd64/$(BINARY_NAME) $(BUILD_DIR)/windows-amd64/$(BINARY_NAME).exe
+
+.PHONY: macos-universal ## Creates macOS universal binary
+macos-universal: clean lint test $(BUILD_DIR)/macos-universal/$(BINARY_NAME)
```

Comment:
`macos-universal` should not depend on `clean`. `macos-universal` is a depedency of `make release`, and the `clean` would remove the binaries created by `cross`.